src/types/schema.d.ts

```diff
@@ -11,7 +11,7 @@ export interface Profile {
   availability_description?: string;
   expected_bar_date?: string; // timestamp
   location: string; // city, ST
-  phone_number: string;
+  phone_number?: string;
```

Comment:
during the last NPO call, Ambreen mentioned she preferred having the phone number field be required for onboarding. Could we go ahead and update this?

Reply:
roger that 🫡